### PR TITLE
Beginning logic for DB update function

### PR DESCRIPTION
**Pull request by creitz**
This branch includes the code logic outlines for updating the DB. It takes articles (not yet retrieved by Watson Discovery) and appends them to a history in the DB for each company.

### CODE REVIEW

File: update/stock_update.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+//---CONFIGURE AUTHENTICATION HERE--------------------
+var CLOUDANT_KEY      = 'XXXXXXXXX';
```
Comment (stevemar) [inline update/stock_update.js:2]: this is fine for first pass but it should ideally be read from a file, like here: https://github.com/IBM/watson-review-analyzer/blob/master/app.js#L15-L26

```
config = jsonfile.readFileSync( __dirname + '/config.json' );

app.use(function (req, res, next){
  req.config = config;
  next();
});

var discovery = new DiscoveryV1({
  username: config.discoveryUsername,
  password: config.discoveryPassword,
  version_date: '2017-11-07'
});
```

Comment (stevemar) [reply update/stock_update.js:2]: you'll want to include arguments for disco and cloudant

**Review by stevemar — APPROVED** [review]
this looks fine to me, feel free to merge it and iterate as you go

File: update/stock_update.js (hunk 1 above)
Comment (creitz) [reply update/stock_update.js:2]: Yeah, thanks. The plan was to move to environment vars but that'll work too.

Comment (creitz) [reply update/stock_update.js:2]: That way the "user" could config them in the bluemix console and not have to edit any code.

Comment (stevemar) [reply update/stock_update.js:2]: it's entirely do-able to do both, this way you can satisfy a local test run and a more production style run. Check env vars first, if they are there, use em. If not, check config file, else, bomb out.

**Pull request merged**